README.md

```diff
@@ -7,6 +7,11 @@
 [![License](https://img.shields.io/cocoapods/l/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![Platform](https://img.shields.io/cocoapods/p/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 
+## DISCLAIMER
+
+*This project is in a closed beta,
```

Comment:
suggestion: how about throwing in some warning emojis to get folks' attention? Ex: ⚠️  ⚠️

Reply:
Woops I merged before I read this comment haha